planner:
  ros__parameters:
    frame_id: "map"
    occupied_cloud_topic: "/octomap_point_cloud_centers"
    grid_res: 0.20
    inflation: 0.20
    bbox_margin: 1.0
    start: [10.0, 0.0, 2.5]
    goal:  [36.63155746459961, -60.41566467285156, 3.5]
    drone_mass : 0.032
    sample_rate : 0.05
    avg_speed : 0.5

sfc_generator:
  ros__parameters:
    frame_id: "map"
    occupied_cloud_topic: "/octomap_point_cloud_centers"
    grid_res: 0.20
    inflation: 0.30
    min_half: [0.4, 0.4, 0.25]
    max_half: [1.5, 1.5, 0.8]
    grow_step: 0.10
    path_stride: 3
goal_manager:
  ros__parameters:
    mode: "goal_only"          # ou "two_click"
    planner_node: "/planner"
    clicked_topic: "/clicked_point"
    default_start: [10.0, 0.0, 2.5]
    use_fixed_z: true
    fixed_z: 0.5

